semgrep/semgrep/config_resolver.py

```diff
@@ -35,16 +38,8 @@
 
 logger = logging.getLogger(__name__)
 
-IN_DOCKER = "SEMGREP_IN_DOCKER" in os.environ
-IN_GH_ACTION = "GITHUB_WORKSPACE" in os.environ
-
-SRC_DIRECTORY = Path(os.environ.get("SEMGREP_SRC_DIRECTORY", Path("/") / "src"))
-OLD_SRC_DIRECTORY = Path("/") / "home" / "repo"
```

Comment:
This has been deprecated for 10.5 months at this point (https://github.com/returntocorp/semgrep/pull/1192), I think we're safe to remove it.